config.js

```diff
@@ -15,6 +15,7 @@ System.config({
 
 System.config({
   "map": {
+    "aurelia-path": "github:aurelia/path@0.9.0",
```

Comment:
aurelia-path is currently at 0.8.1. I'm assuming it'll be bumped to 0.9.0 with the API addition from aurelia/path#13.